README.md

```diff
@@ -68,8 +68,15 @@ Python helpers to comunicate with Taskwarrior and Trello.
 You can use your package system to install it, but the best way is to use
 a Python 3 virtualenv:
 
+To ensure that this is a python3 env, first locate your python3 binary:
+```sh 
+$ which python3
+/usr/bin/python3
+```
+then use that binary to ensure that the correct version of python is used (if your result is different to the above, change it in this code):
+
 ```sh
-virtualenv trw
+virtualenv -p /usr/bin/python3 trw
```

Comment:
The option `-p /usr/bin/python3` can be used always. Is good idea add at this point, but all the previous explanation not is neccesary.

Dont worry I just add it to readme. Thanks for suggestion!

Reply:
that's cool - just wanted to see if I could put something helpful in.
Thanks :+1: